### ISSUE TITLE
Empty initial values when using comma as decimal separator

### ISSUE THREAD

**Issue by Patrik-Lundqvist** [uhlryk/react-dynamic-number#22]
If init model value is 0.5 and comma is used as view decimal separator, the input will be empty.

**Comment by uhlryk** [comment]
Thanks, for reporting this bug. I reproduced it and therefore I will fix it soon.

**Comment by uhlryk** [comment]
should be fixed in newest version, please check it and confirm if it is ok.

**Comment by Patrik-Lundqvist** [comment]
Nice work! I checked it but it still need some fixing. The init value now shows but if the underlying value changes, the display value will be 0.

This can be replicated by changing the last input value in the example to "0,5" and checking the input above. Also, the value for the last example are bound to the wrong onChange value which makes this test possible 😄

![image](https://cloud.githubusercontent.com/assets/5230331/19232025/b8c56da6-8edd-11e6-8460-dbf0a335a1fd.png)

I noticed there's an accidental console.log left in there too 😉
